src/EventSubscriber/ShowGeneratedFiles.php

```diff
@@ -0,0 +1,89 @@
+<?php
+
+/**
+ * @file
+ * Contains \Drupal\AppConsole\EventSubscriber\ShowGeneratedFiles.
+ */
+
+namespace Drupal\AppConsole\EventSubscriber;
+
+use Drupal\AppConsole\Command\Helper\TranslatorHelper;
+use Symfony\Component\Console\ConsoleEvents;
+use Symfony\Component\Console\Event\ConsoleTerminateEvent;
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+
+class ShowGeneratedFiles implements EventSubscriberInterface
+{
+
+    /**
+     * @var TranslatorHelper
+     */
+    protected $trans;
+
+    /**
+     * @var string
+     */
+    protected $completedMessageKey = 'application.console.messages.completed';
+
+    /**
+     * @param TranslatorHelper $trans
+     */
+    public function __construct(TranslatorHelper $trans)
+    {
+        $this->trans = $trans;
+    }
+
+
+    /**
+     * @param ConsoleCommandEvent $event
+     */
+    public function showGeneratedFiles(ConsoleTerminateEvent $event)
+    {
+        /** @var \Drupal\AppConsole\Command\Command $command */
+        $command = $event->getCommand();
+        $output = $event->getOutput();
+
+        if ($event->getExitCode() != 0) {
+            return;
+        }
+
+        $completedMessageKey = 'application.console.messages.completed';
+
+        if ('self-update' == $command->getName()) {
+            return;
+        }
+
+        if (method_exists($command, 'getMessages')) {
```

Comment:
In this comparation I think that is better ask if is an instance of Drupal\AppConsole\Command\Command, what do you think about? @jmolivas